conf/modules.config

```diff
@@ -463,8 +463,10 @@ process {
             "--sample_id_col \"${params.observations_id_col}\"",
             "--minimum_samples ${params.filtering_min_samples}",
             "--minimum_abundance ${params.filtering_min_abundance}",
+            "--minimum_proportion_not_na ${params.filtering_min_proportion_not_na}",
```

Comment:
This needs conditional treatment as well, right?

Reply:
No, there is a default value for that param in the nextflow.config. If it is provided together with the min sample number, the latter will be used in the module

Reply:
What if someone sets the value to null?

Reply:
Ah, fair enough!